test/test_gaussian.py

```diff
@@ -297,6 +297,41 @@ def test_eager_subs_variable():
     assert g2.precision is g1.precision
 
 
+@pytest.mark.parametrize('subs', [
+    (('x', 'Variable("u", reals()) * 2'),),
+    (('y', 'Variable("v", reals(4)) + 1'),),
```

Comment:
nit: Could you add the following test case
```(('y', 'Einsum("abc,bc->a", (Tensor(torch.randn(4, 3, 5)), Variable("v", reals(3, 5))))'),),```
which is useful to verify dim stuffs in affine sub?

Reply:
done.